Providers/Pocket/PocketFeed.swift

```diff
@@ -113,7 +113,9 @@ class Pocket: PocketStoriesProviding, FeatureFlaggable, URLCaching {
     }
 
     private var shouldUseMockData: Bool {
-        return featureFlags.isCoreFeatureEnabled(.useMockData) && (pocketKey == "" || pocketKey == nil)
+        guard let pocketKey = pocketKey else { return false }
```

Comment:
If `pocketKey` is nil we have to check if the feature flag is on before deciding what to return here. 

```guard let pocketKey = pocketKey else { return featureFlags.isCoreFeatureEnabled(.useMockData) ? true : false }```

Reply:
Brain fart. yes. totally.